deploy/secrets-store-csi-driver.yaml

```diff
@@ -42,7 +42,7 @@ spec:
             - name: registration-dir
               mountPath: /registration
         - name: secrets-store
-          image: docker.io/deislabs/secrets-store-csi:v0.0.5
+          image: aramase/secrets-store-csi:v0.0.7
```

Comment:
should we update this image tag after we cut a new release of the csi driver

Reply:
Yeah, we should do it after we cut a new release.

Reply:
Done!